packages/common/src/utils.ts

```diff
@@ -93,6 +96,33 @@ function parseGethParams(json: any) {
     throw new Error('nonzero terminal total difficulty is not supported')
   }
 
+  let customHardforks: HardforksDict | undefined = undefined;
+  if (config.blobSchedule !== undefined) {
+    customHardforks = {};
+    const blobGasPerBlob = 131072;
+    for (const [hfKey, hfSchedule] of Object.entries(config.blobSchedule)) {
+      const hfConfig = hardforksDict[hfKey];
+      if (hfConfig === undefined) {
+        throw new Error(`unknown hardfork=${hfKey} specified in blobSchedule`);
+      }
+      const { target, max, baseFeeUpdateFraction: blobGasPriceUpdateFraction } = hfSchedule as { target?: number, max?: number, baseFeeUpdateFraction?: undefined };
+      if (target === undefined || max === undefined) {
+        throw new Error(`undefined target or max specified in blobSchedule for hardfork=${hfKey}`);
+      }
+
+
+      // copy current hardfork info to custom and add blob config
+      const customHfConfig = JSON.parse(JSON.stringify(hfConfig));
+      customHfConfig.params = {
+        ...customHardforks.params,
+        // removes blobGasPriceUpdateFraction key to prevent undefined overriding if undefined
```

Comment:
I don't think this works, if `blobGasPriceUpdateFraction` is undefined then it will set the param to `undefined` also.